.github/workflows/serverless-pre-release-qa.yml

```diff
@@ -0,0 +1,139 @@
+# Workflow to create a new pre-release with qa suffix
+
+name: Pre-release QA
+
+# Controls when the action will run.
+on:
+  workflow_dispatch:
+    inputs:
+      tag-prefix:
+        description: Release tag name.
+        required: true
+      title:
+        description: Release title - The title that will be given to this pre-release.
+        required: true
+
+jobs:
+  build-and-deploy:
+    # TODO: remove branch tag (@gian_CHI-1263)
+    uses: techmatters/serverless/.github/workflows/deploy_development_e2e_end_to_end.yml@gian_CHI-1263
+    secrets:
+      AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+      AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+      AWS_DEFAULT_REGION: ${{ secrets.AWS_DEFAULT_REGION }}
+      E2E_DEV_ACCOUNT_SID: ${{ secrets.E2E_DEV_ACCOUNT_SID }}
+      E2E_DEV_AUTH_TOKEN: ${{ secrets.E2E_DEV_AUTH_TOKEN }}
+    with: 
+      send-slack-message: 'false'
+  run-e2e-tests:
+    needs: build-and-deploy
+
+    runs-on: ubuntu-latest
+
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v2
+      - name: Use Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: '14.x'
+
+      # Clone flex-plugins repo
+      - name: Clone flex-plugins repo
+        run: git clone https://github.com/techmatters/flex-plugins
+        shell: bash
+      # Build Playwright
+      - name: Install e2e-tests dependencies
+        run: npm install
+        working-directory: ./flex-plugins/e2e-tests
+      - name: Setup dependencies for playwright/browsers
```

Comment:
Perhaps we could use the Playwright docker image to shortcut this part of the step?

But as I said on Slack, it might be worth publishing the entire E2E tests as a docker image for use in this and other workflows in the future, which would shortcut this whole step

Either way, not for changing in this PR obvs

Reply:
I agree with this :raised_hands: